README.rst

```diff
@@ -35,6 +36,15 @@ Documentation
 You can read more about `installing Flocker`_, follow a `tutorial`_ and learn about the `features of Flocker and its architecture`_ in the docs.
 
 
+Support
```

Comment:
This section overlaps / contradicts the line a few lines above:

>  If you have any issues or feedback, you can talk to us

That links to the talk to us page.